internal/rpmmd/repository.go

```diff
@@ -150,6 +156,39 @@ func LoadRepositories(confPaths []string, distro string) (map[string][]RepoConfi
 		return nil, err
 	}
 
+	if strings.HasPrefix(distro, "rhel-8") {
+		var sslClientKey string
+		var sslClientCert string
+
+		regexKey := regexp.MustCompile("^[0-9]*-key.pem")
+		regexCert := regexp.MustCompile("^[0-9]*.pem")
+
+		entitlementFiles, err := ioutil.ReadDir("/etc/pki/entitlement/")
+		if err != nil {
+			return nil, &RepositoryError{"LoadRepositories failed: unable to read /etc/pki/entitlement/"}
+		}
+		for _, file := range entitlementFiles {
+			matchKey := regexKey.MatchString(file.Name())
+			if matchKey {
+				sslClientKey = path.Join("/etc/pki/entitlement/", file.Name())
+				continue
+			}
+
+			matchCert := regexCert.MatchString(file.Name())
+			if matchCert {
+				sslClientCert = path.Join("/etc/pki/entitlement/", file.Name())
+			}
+		}
+
+		for arch, repoConfigs := range repos {
+			for i := range repoConfigs {
+				repos[arch][i].IgnoreSSL = false
```

Comment:
Why are you overriding this here again?

Reply:
I have now switched IgnoreSSL to be SSLVerify to better match the fields supported by dnf and this section of code has been changed.